src/Components/Main/Pages/ChatPage/ChatMessage/ChatMessage.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ReactElement } from 'react';
+import { memo } from 'react';
+import type { IChatMessage } from '../../../../../utils/types/api';
+import userPhoto from '../../../../../assets/images/user.png';
+import classes from './ChatMessage.module.scss';
+
+type PropsType = {
+  message: IChatMessage;
+};
+
+export const ChatMessage = memo<PropsType>(({ message }): ReactElement => {
+  let time = 'no time';
+  let text = message.message;
+
+  if (text.includes('UTC+0')) {
+    time = text.split(' ').slice(-2).join(' ');
+    text = text.split(' ').slice(0, -2).join(' ');
+  }
```

Comment:
What are you trying to achieve here? =)
It doesnt seem reliably to check for 'UTC+0' =)

Reply:
The .getUTCHours() and .getUTCMinutes() methods themselves return the time in UTC format, I do not need to convert the time to UTC.

Reply:
The server does not return the time...
But if you use my application, then you send a UTC time string which includes 'UTC+0' to the server (The ChatFormContainer.tsx file). For users who send messages from other places I do not expect 'UTC+0', and accordingly I add 'no time' instead of the time.
This is all expected behavior:)

Reply:
```suggestion
  if (text.includes('UTC+0')) {
    time = text.split(' ').slice(-2).join(' ');
    text = text.split(' ').slice(0, -2).join(' ');
  }
```

Reply:
```
  let time = 'no time';
  let text = message.message;

  if (text.includes('UTC+0')) {
    time = text.split(' ').slice(-2).join(' ');
    text = text.split(' ').slice(0, -2).join(' ');
  }
```